_rules/SC4-1-2-iframe-unique-name.md

```diff
@@ -0,0 +1,136 @@
+---
+name: IFrame has a unique accessible name
+
+description: |
+ Each iframe element has a unique accessible name
+
+success_criterion:
+- 4.1.2
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to `iframe` elements within a given [document context](#document-context) that have either `title`, `aria-label` or `aria-labelled-by` attribute.
```

Comment:
I suggest changing this to say "iframes with an accessible name". And than follow it by listing the attributes. We could also change this here to say non-empty accessible name. WDYT?

Reply:
Note: 

- add exposed to assistive technologies.